doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -240,10 +240,15 @@ nRF Desktop
     * Set the max compiled-in log level to ``warning`` for the Non-Volatile Storage (:kconfig:option:`CONFIG_NVS_LOG_LEVEL`).
     * Lowered a log level to ``debug`` for the ``Identity x created`` log in the :ref:`nrf_desktop_ble_bond`.
 
-  * Removed separate configurations enabling :ref:`zephyr:shell_api` (:file:`prj_shell.conf`).
-    Shell support can be enabled for a given configuration with the single Kconfig option (:ref:`CONFIG_DESKTOP_SHELL <config_desktop_app_options>`).
   * By default, nRF Desktop dongles use the Bluetooth appearance (:kconfig:option:`CONFIG_BT_DEVICE_APPEARANCE)` of keyboard.
     The new default configuration value improves consistency with the used HID boot interface.
+  * Updated the :ref:`nrf_desktop_fast_pair_app` to remove the Fast Pair advertising payload for the dongle peer.
+
+* Removed:
+
+  * Removed Fast Pair discoverable advertising payload on Resolvable Private Address (RPA) rotation during discoverable advertising session.
+  * Removed separate configurations enabling :ref:`zephyr:shell_api` (:file:`prj_shell.conf`).
```

Comment:
```suggestion
  * Fast Pair discoverable advertising payload on Resolvable Private Address (RPA) rotation during discoverable advertising session.
  * Separate configurations enabling :ref:`zephyr:shell_api` (:file:`prj_shell.conf`).
```

Reply:
Addressed. You can take a look at diff from the last force push:

https://github.com/nrfconnect/sdk-nrf/compare/6cf784ddb2d3587ba25ebd267cba074fc6c57431..9a71786ea31bc36ce3a7249d7252b98292393a42